index.html

```diff
@@ -178,20 +270,23 @@ <h2 class="primary-text">About me</h2>
           If you like what you see and have a project you need coded, 
           don’t hestiate to contact me.</p>
           <button type="button">Get My Resume</button>
+          <hr>
       </section>   
   
-      <section class="skills-container skills-category">
-        <hxr class="skils-title">
-          <img src="assests/images/diomand.png" alt="Diomand shape">
-          <h3 class="language">Language</h3>
-        </hxr>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-  
+      <section class="skills-container">
+        <section skills-category>
+          <hxr class="skils-title">
+            <img src="assests/images/diomand.png" alt="Diomand shape">
+            <h3 class="language">Language</h3>
+          </hxr>
+            <ul>
```

Comment:
- [ ] According to the project requirement you should stick to the design as much as possible (e.g., font, colors, images, text, margins). Please use the same color, if you see your design then the color is different in that 2 sections.
- [ ] Please remove the line as well there is 2 line
### Your design
![his](https://user-images.githubusercontent.com/81727988/199897697-5eaa1ef1-9345-43d7-bb27-4903cfcfa997.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199897733-8d41d4ec-1a11-40cf-9348-45e36189f154.png)